src/components/Views/Home/AboutUs/AboutUs.jsx

```diff
@@ -6,55 +6,55 @@ import AboutStats from "./AboutStats";
 const AboutUs = () => {
   return (
     <section
-      name='about'
-      id='about-us'
-      className='my-20 sm:my-0 sm:py-20 scroll-mt-24 sm:scroll-mt-10'
+      name="about"
```

Comment:
Extra changes because of inverted commas. Change your VS code prettier config that it uses single inverted commas not the double so that this extra change can be removed. @ebaadwaseem